AdvancedProgramming2.csproj

```diff
@@ -1,8 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net48</TargetFramework>
```

Comment:
Just download and install dotnet 7

Reply:
i did download it but i made the pull request so other student won't have to download any thing and just clone the repo